tox.ini

```
@@ -41,7 +41,7 @@ commands =
   python -c '\
   import os, subprocess, sysconfig;\
   os.environ["LD_LIBRARY_PATH"] = os.pathsep.join(filter(None, [sysconfig.get_config_var("LIBDIR"), os.getenv("LD_LIBRARY_PATH")]));\
-  subprocess.run(["cargo", "test", "--no-default-features"])'
+  subprocess.run(["cargo", "test", "--no-default-features"]).check_returncode()'
```

Not that it particularly matters in this context. But what I normally do is:

```suggestion
  subprocess.run(["cargo", "test", "--no-default-features"], check=True)'
```

---

I think pylint has a lint against failing to check the return code of `subprocess.run` - it might be a motivation to move this inline script into a proper file in the `tools` directory so it's subject to lint?

---

It does: https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/subprocess-run-check.html

---

Seems like a good idea, and should let us use the same script from tox and CI.

Done in 663eaa8.